akka-sse/src/main/scala/de/heikoseeberger/akkasse/ServerSentEvent.scala

```diff
@@ -55,14 +55,20 @@ object ServerSentEvent {
     m |= m >> 16
     m + 1
   }
+
+  /**
+   * A comment event used for sending heartbeats to the client.
+   */
+  final val Heartbeat = ServerSentEvent("heartbeat", comment = true)
 }
 
 /**
- * Reprsentation of a Server-Sent Event.
+ * Representation of a Server-Sent Event.
  * @param data data which may span multiple lines
  * @param eventType optional event type, must not contain \n or \r
+ * @param comment determines if this event is a comment, i.e. has no field name
  */
-final case class ServerSentEvent(data: String, eventType: Option[String] = None) {
+final case class ServerSentEvent private[akkasse] (data: String, eventType: Option[String] = None, comment: Boolean = false) {
```

Comment:
This makes the whole event a comment. I'd rather suggest an approach also supporting the various fields of the spec, e.g. "retry" and "id".